lib/src/flare_actor_component.dart

```diff
@@ -0,0 +1,35 @@
+part of flame_flare;
+
+class _FlareActorComponentPipelineOwner extends PipelineOwner {}
+
+/// A [PositionComponent] that renders a [FlareActorAnimation]
+class FlareActorComponent extends PositionComponent {
+  final FlareActorAnimation flareAnimation;
+
+  FlareActorComponent(this.flareAnimation);
+
+  @override
+  void onMount() {
```

Comment:
Add a @mustCallSuper here, in case someone needs to override this, we need to make sure that the super is called

Reply:
wilco